docs/management_api_v2.yml

```
@@ -215,6 +215,42 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+    delete:
+      operationId: Delete Releases
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: Delete the releases with given names
+      description: |
+        Deletes releases with names provided in the message body.
+        Releases used by deployments in progress can not be deleted
+        until deployment finishes.
+      produces:
+        - application/json
+      parameters:
+        - name: delete_request
+          in: body
+          required: true
+          type: object
+          properties:
+            artifact_names:
+              type: array
+              description: List of names of artifacts to delete.
+              items:
+                type: string
```

```suggestion
      parameters:
        - name: delete_request
          in: body
          required: true
          type: object
          properties:
            release_names:
              type: array
              description: List of names of releases to delete.
              items:
                type: string
```

to stick to the release focus or maybe rather:

```
      parameters:
        - name: name
          in: query
          description: Name of the release to be deleted
          required: true
          type: string
```

to align with the existing filtering structure when listing releases (so `DELETE /deployments/releases?name=foo&name=bar` along with `GET /deployments/releases?name=baz&name=icecream`)

OTOH you likely know other reasons not to use the query params for that 😅

---

I'm fine with using query params here; thanks